cmake/modules/PyTypeUtils.cmake

```diff
@@ -41,6 +41,11 @@ add_custom_target(
   ${ALL_TESTS_TARGET}
 )
 
+string(COMPARE EQUAL ${CMAKE_BUILD_TYPE} "Debug" is_debug_build)
```

Comment:
Could this be replaced with `if(${CMAKE_BUILD_TYPE} STREQUAL "Debug")`?

Reply:
I have sent an internal CL to fix this. However, can you add a commit to this PR and let copybara import it? The fix you suggested should also work.

Reply:
If you are going to attempt this, you will need quotes like this: "${CMAKE_BUILD_TYPE}". I preferred "string(COMPARE ...)" over the "if(...)" because CMake is what it is: https://cmake.org/pipermail/cmake/2013-February/053587.html. The suggested alternate there also has problems. The issue is that "if" is not a statement but a function which needs to be matched by another function "endif"!

CMake also has an expectation that we write it this way:

if("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
...
endif("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")

So, as a practice, I have been using "string(COMPARE ...)" followed by an "if(...)".

Reply:
Just noticed that you have actually assigned this to me. I am not sure what that means, but as I had asked earlier, can I push a commit to this branch (and expect copybara to do the right import?)